cnf-tests/Dockerfile.openshift

```diff
@@ -1,5 +1,5 @@
 # This dockerfile is specific to building the OpenShift CNF stresser image
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.22-openshift-4.18 as builder-stresser
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.19 as builder-stresser
```

Comment:
base on the ART automation 4.19 is base on 1.23 https://github.com/openshift/sriov-network-operator/blob/master/Dockerfile.sriov-network-config-daemon.ocp#L1

can you please update the images?